src/xdocs/config_filters.xml

```diff
@@ -991,6 +991,215 @@ public class InputTest {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathSingleFilter">
+      <subsection name="Description" id="SuppressionXpathSingleFilter_Description">
+        <p>Since Checkstyle 8.16</p>
+        <p>
+          <code>SuppressionXpathSingleFilter</code> is a wrapper of <code>XpathFilter</code>,
+          which contains xpath-expressions that are queries for suppressed nodes inside the
+          AST tree. This filter could be used directly in the configuration file. It is for
```

Comment:
I question everything after `which` and this being in the first sentence. It seems very technical and not a friendly description of the filter.

`XpathFilter` isn't visible to users, `SuppressionXpathFilter` is so it would make more sense to users to mention that filter instead of internal one.
Please also explain that this filter only supports 1 suppression, and will need multiple instance if wants to suppress multiple violations.